test/tests/primitives/op/number.js

```diff
@@ -262,13 +262,13 @@ describe('Numbers', function(){
 
     describe('range', function(){
         it('should work', TestCheddarFrom.Code(
-            'print "1:5=" + String::(1:5)',
```

Comment:
you forgot to make the `String::(1:5)` -> `String::(1|>5)` (same thing below too) ;)